src/components/mui/table/mui-table.js

```diff
@@ -126,13 +98,11 @@ const MuiTable = ({
                       <TableSortLabel
                         active={sortCol === col.columnKey}
                         direction={
-                          sortCol === col.columnKey
-                            ? sortDir === 1
-                              ? "asc"
-                              : "desc"
+                          sortCol === col.columnKey && sortDir === "-1"
+                            ? "desc"
                             : "asc"
                         }
-                        onClick={() => onSort(_, col.columnKey, sortDir * -1)}
+                        onClick={() => onSort(col.columnKey, sortDir * -1)}
```

Comment:
@santipalenque here u multiple by -1 ( int ) but on the check u use a string 
https://github.com/fntechgit/summit-admin/pull/642/files#diff-3cdfde7e970b4b68cea921a81e31188807ed5e45b7c29cef62bdb30e4c5e898dR101